tests/test_files/html/images.html

```diff
@@ -0,0 +1,7 @@
+<h1>Images should be removed</h1>
```

Comment:
Did we need to actually reference an image in this example or is this OK?

Reply:
Each line isn't a standalone example.  That's just for readability when looking at the markdown file, which has h1 and h2 elements.